README.md

```diff
@@ -137,3 +137,33 @@ In particular this means that this project may not depend on GPL-licensed or AGP
 as these would violate the terms of those libraries' licenses.
 
 The contents of this repository are protected by Crown Copyright (C).
+
+## Performance
+The performance of PS Adaptor was tested with JMeter tool. The use case that was used was the simulation of the patient transfer request
+where we were sending Ehr records data to PS Adaptor which was expected to create a bundle of it. We were looking to observe how PS Adaptor 
```

Comment:
Maybe the whole thing should read more: 
"The performance of PS Adaptor was tested with JMeter tool. The use case tested was the simulation of the patient transfer request.  This was tested by sending EHR record requests to the PS Adaptor, and we expected to receive a bundle back.  We used these tests to observe how the PS Adaptor handles a heavy workload and to profile the CPU and memory usage during the testing process.

Reply:
thanks, looks better.